web-app/src/main/webapp/css/style.css

```diff
@@ -29,6 +28,93 @@
   height: 3em;
 }
 
+/* Options menu */
+#menu {
+  background-color: rgba(0, 0, 23, 0.8);
+  color: #e0e0e0;
+  height: 100vh;
+  position: absolute;
+  width: 600px;
+}
+
+#close-menu {
+  background-image: url(/images/back-arrow.svg);
+  background-repeat: no-repeat;
+  background-size: 70px 70px;
+  height: 70px;
+  margin-left: 20px;
+  margin-top: 20px;
+  width: 70px;
+}
+
+#auth-container {
+  font-size: 1.75em;
+  padding-right: 20px;
+  padding-top: 40px;
+}
+
+#auth-container a {
+  color: #e0e0e0;
+  text-decoration: none;
+}
+
+#menu-title {
+  font-size: 2.7em;
+  padding: 0 60px;
+  text-align: left;
+}
+
+.menu-subheading {
+  font-size: 2.3em;
+  padding-right: 40px;
+  text-align: right;
+}
+
+.right {
+  text-align: right;
```

Comment:
I think this UX pattern is quite rare. Could we use left-align but use a separator? https://material.io/components/navigation-drawer